inference-engine/samples/speech_sample/main.cpp

```diff
@@ -857,7 +857,10 @@ int main(int argc, char *argv[]) {
 
                 uint32_t numFramesReference(0), numFrameElementsReference(0), numBytesPerElementReference(0),
                         numBytesReferenceScoreThisUtterance(0);
-                const uint32_t numScoresPerFrame = ptrOutputBlob.size() / batchSize;
+                auto dims = outputs.empty() ? cOutputInfo.rbegin()->second->getDims() : cOutputInfo[outputs[next_output]]->getDims();
+                const uint32_t numScoresPerFrame = std::accumulate(std::begin(dims), std::end(dims), static_cast<size_t>(1), std::multiplies<size_t>());
```

Comment:
please use size_t{1} instead of static cast.
You can also use auto for declaring numScoresPerFrame  variable